backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -74,7 +75,8 @@ def resolve_fetch_failures_0_members(
      small performance gain."""
     print("Resolving fetch failures: ostensibly new (possibly draft) concept sets with >0 expressions but 0 members")
     # Collect failures
```

Comment:
## Data fix: "Success result: No members after 2 hours. Considering resolved."
We had previously set up this script to run for 2 hours and, if no members were detected, we were considering it to be the case that there was simply no expansion; that there were 0 members. However, I believe that this erroneously did not check to see if those csets were drafts, and thus erroneously marked them as being completed.

I solved by running a query to remove the "success" part of the comment string and set `success_datetime` to NULL. Then I fetched data for all of these csets.